README.md

```diff
@@ -20,11 +20,53 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install volabit
+    $ git clone https://github.com/coincovemx/api-client.git
+    $ cd api-client
+    $ bundle
+    $ gem build volabit.gemspec
+    $ gem install volabit-[version].gem
 
 ## Usage
 
-TODO: Write usage instructions here
+Instance a new Volabit client object.
+
+```ruby
+app_id      = 'The registered API for your APP.'
+secret      = 'The registered secret for your APP.'
+callback    = 'The registered callback URL for your APP'
+environment = 'The environment to work on (production or test)'
+
+client = Volabit::Client.new(app_id: app_id, app_secret: secret, app_url: callback, environment: environment)
```

Comment:
I've removed the named params but retained the option to define the working environment upon instantiation as optional, so the following code fragments do the same:

```ruby
# Environment set on client creation.
# (Optional and defaults to 'production' if none provided).
volabit_client = Volabit::Client.new YOUR_APP_ID, YOUR_SECRET, CALLBACK, ENV
```

```ruby
# Environment toggled after client instantiation.
volabit_client = Volabit::Client.new YOUR_APP_ID, YOUR_SECRET, CALLBACK
volabit_client.sandbox true
```
How about it?